tools/linter/adapters/ufmt_linter.py

```diff
@@ -29,7 +29,6 @@
                 fnmatch.translate,
                 [
                     # **
-                    "**",
```

Comment:
Why are you removing this line? Looks like it would exclude a lot of files from UFMT